platform-metrics/src/main/java/org/hypertrace/core/serviceframework/metrics/PlatformMetricsRegistry.java

```diff
@@ -193,6 +238,8 @@ public synchronized static void initMetricsRegistry(String serviceName, Config c
         case "testing":
           initTestingMetricsReporter();
           break;
+        case "pushgateway":
```

Comment:
@inespo I think we shouldn't let any job/service to add both `prometheus` and `pushgateway` reporters because that would cause two instances of `PrometheusMeterRegistry` and I don't know the full side effects of that. Should we add a validation for that?

Reply:
Good point. I'll add a validation.